cmd/bootkube/render.go

```diff
@@ -24,7 +24,7 @@ const (
 	bootEtcdOffset               = 20
 	defaultServiceBaseIP         = "10.3.0.0"
 	defaultEtcdServers           = "https://127.0.0.1:2379"
-	defaultSelfHostedEtcdServers = "http://127.0.0.1:2379"
+	defaultSelfHostedEtcdServers = "https://127.0.0.1:2379"
```

Comment:
should just remove this and use `defaultEtcdServers` now that they are the same.